packages/@expo/metro-config/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### 💡 Others
 
+- Shim web streams when bundling for the server. ([#36407](https://github.com/expo/expo/pull/36407) by [@EvanBacon](https://github.com/EvanBacon))
```

Comment:
This was a bit confusing to me. We shim this in all environments but the server, right?

Reply:
It's never added in web. It's just for native client bundles.

Reply:
```suggestion
- Shim web streams API when bundling for native client environments. ([#36407](https://github.com/expo/expo/pull/36407) by [@EvanBacon](https://github.com/EvanBacon))
```